whatlies/embeddingset.py

```diff
@@ -679,6 +679,119 @@ def plot(
         )
         return self
 
+    def plot_3d(
+        self,
+        x_axis: Union[int, str, Embedding] = 0,
+        y_axis: Union[int, str, Embedding] = 1,
+        z_axis: Union[int, str, Embedding] = 2,
+        color: str = None,
+        axes_metric: Optional[Union[str, Callable, Sequence]] = None,
+        annot: bool = True,
+    ):
+        """
+        Creates a 3d visualisation of the embedding.
+
+        Arguments:
+            x_axis: the x-axis to be used, must be given when dim > 3; if an integer, the corresponding
+                dimension of embedding is used.
+            y_axis: the y-axis to be used, must be given when dim > 3; if an integer, the corresponding
+                dimension of embedding is used.
+            z_axis: the z-axis to be used, must be given when dim > 3; if an integer, the corresponding
+                dimension of embedding is used.
+            color: the property to user for the color
+            axes_metric: the metric used to project each embedding on the axes; only used when the corresponding
+                axis is a string or an `Embedding` instance. It could be a string (`'cosine_similarity'`,
+                `'cosine_distance'` or `'euclidean'`), or a callable that takes two vectors as input and
+                returns a scalar value as output. To set different metrics for different axes, a list or a tuple of
+                the same length as `axes` could be given. By default (`None`), normalized scalar projection (i.e. `>` operator) is used.
```

Comment:
"To set different metrics for each of the three axes, a list or tuple of three elements could be given."

The current one is the one used for `plot_interactive_matrix`.